README.md

```diff
@@ -20,51 +20,58 @@ First since we'll launch kind with GPU support, ensure that the following prereq
    can be done by following the instructions
    [here](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html).
 1. Configure the NVIDIA Container Runtime as the **default** Docker runtime:
-   ```bash
+   ```console
    sudo nvidia-ctk runtime configure --runtime=docker --set-as-default
    ```
 1. Restart Docker to apply the changes:
-   ```bash
+   ```console
    sudo systemctl restart docker
    ```
 1. Set the `accept-nvidia-visible-devices-as-volume-mounts` option to `true` in
    the `/etc/nvidia-container-runtime/config.toml` file to configure the NVIDIA
    Container Runtime to use volume mounts to select devices to inject into a
    container.
 
```

Comment:
While you're changing things do you want to add this command below this step as well:
```
sudo nvidia-ctk config --set accept-nvidia-visible-devices-as-volume-mounts=true --in-place
```